.gitignore

```diff
@@ -331,4 +331,5 @@ ASALocalRun/
 **/.ipynb_checkpoints/**
 
 # No need to store DeployedQueries.json - it is auto generated anyway
```

Comment:
Did you mean to make this change?  If not, it may be due to not merging the latest master before you pushed your changes up for PR.

Reply:
May be due to merging